recipes/liblsl/meta.yaml

```
@@ -0,0 +1,44 @@
+# Conda build recipe for liblsl
+
+{% set name = "liblsl" %}
+{% set version = "1.14.1b8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sccn/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 196d28608f0c91fb589fd6392e5f4bf2c4d6d5ee4a68ae1df22bf7352a7621b4
+
+{% set common_cmake_args = "-DCMAKE_BUILD_TYPE=Release -DLSL_BUNDLED_PUGIXML=OFF -DLSL_UNIXFOLDERS=ON -S . -B build" %}
+
+build:
+  number: 0
+  script:
+  - 'cmake $CMAKE_ARGS {{ common_cmake_args }}'  # [not win]
+  - 'cmake -DCMAKE_INSTALL_PREFIX="{{ PREFIX }}" -DCMAKE_PREFIX_PATH="{{ PREFIX }}" {{ common_cmake_args }} -G "%CMAKE_GEN%"'  # [win]
+  - 'cmake --build build --config Release -j --target install'
+
+requirements:
+  build:
+    - cmake >=3.12
+    - {{ compiler('cxx') }}
+    - pugixml
+  host:
```

Library dependencies should go into `host`
```suggestion
  host:
    - pugixml
```

---

Done.